test/ggsn_SUITE.erl

```diff
@@ -963,25 +978,61 @@ error_indication(Config) ->
     ok.
 
 %%--------------------------------------------------------------------
-ipv6_pdp_context_request() ->
-    [{doc, "Check Create PDP Context, Delete PDP Context sequence "
-           "for IPv6 contexts"}].
-ipv6_pdp_context_request(Config) ->
-    {GtpC, _, _} = create_pdp_context(ipv6, Config),
-    delete_pdp_context(GtpC),
+pdp_context_request_bearer_types() ->
+    [{doc, "Create different IP bearers against APNs with restrictions/preferences"}].
 
-    ?equal([], outstanding_requests()),
-    ok = meck:wait(?HUT, terminate, '_', ?TIMEOUT),
-    meck_validate(Config),
-    ok.
+pdp_context_request_bearer_types(Config) ->
+    ok = meck:expect(ergw_gsn_lib, allocate_ips,
+		     fun(AllocInfo, APNOpts, SOpts, DualAddressBearerFlag, Context) ->
+			     try
+				 meck:passthrough([AllocInfo, APNOpts, SOpts, DualAddressBearerFlag, Context])
+			     catch
+				 throw:#ctx_err{} = CtxErr ->
+				     meck:exception(throw, CtxErr)
+			     end
+		     end),
 
-%%--------------------------------------------------------------------
-ipv4v6_pdp_context_request() ->
-    [{doc, "Check Create PDP Context, Delete PDP Context sequence "
-           "for dual stack IPv4/IPv6 contexts"}].
-ipv4v6_pdp_context_request(Config) ->
-    {GtpC, _, _} = create_pdp_context(ipv4v6, Config),
-    delete_pdp_context(GtpC),
+    {GtpC1, _, _} = create_pdp_context({ipv4, false, default}, Config),
```

Comment:
Could these create_pdp_contetext/2 + delete_pdp_context/1
be simplified with a single create_delete_pdp_context/2 that just did both?
As a way to avoid the GtpC1, GtpC2, ...

Reply:
Other tests do things in between those and I would like to keep the structure somewhat uniform.